docs/en/stack/ml/nlp/ml-nlp-model-ref.asciidoc

```diff
@@ -86,6 +154,26 @@ Using `DPREncoderWrapper`:
 * https://huggingface.co/nateraw/bert-base-uncased-emotion
 * https://huggingface.co/cardiffnlp/twitter-roberta-base-sentiment
 
+[discrete]
+=== Text classification expected model output
+
+With text classification, the entire sequence is classified. An example of this is sentiment analysis.
+
+The output of the model must be <number of sequences> <classification score of each sequence>.
```

Comment:
> The output of the model must be <number of sequences> <classification score of each sequence>.

Per my suggestion for the previous sections, I suggest using something like this instead: "It is a nested set of {arrays? objects? tuples?} for each inferred sequence, where every sequence has classification scores for each entity label."